interview-questions.md

```diff
@@ -3,44 +3,52 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. In a model called Animal that has_many Sightings, what is the name of the foreign key? Would the foreign key be part of the Animal model or the Sightings model?
 
-  Your answer:
+  Your answer:  The foreign key should be called animal_id, and it is within Sightings.
 
-  Researched answer:
+  Researched answer: The foreign key should be called animal_id as an integer datatype, and it should be a field within Sightings.
 
 
 
 2. Which routes must always be passed params and why?
 
-  Your answer:
+  Your answer: Routes that have arguments within their path always need an argument or they will throw an error.
 
-  Researched answer:
+  Researched answer:Routes that have arguments within their pathway and if no argument does not have a pathway will throw an error.
 
 
 
 3. Write a rails route for a controller called "main" and a page called "game" that takes in a parameter called "guess".
 
-  Your answer:
+  Your answer: get '/game/:guess' => 'main'
```

Comment:
we'd just need to add a controller method to this route: `/game/:guess => main#game`